test/functional/http/get/node/transactions_unsigned.js

```diff
@@ -271,7 +273,9 @@ describe('GET /api/node', () => {
 						expect(res.body.data).to.not.empty;
 						expect(res.body.data.length).to.be.at.least(numOfTransactions);
 						res.body.data.map(transaction => {
-							expect(transaction.senderId).to.be.equal(senderAccount.address);
+							return expect(transaction.senderId).to.be.equal(
```

Comment:
Basically in es6 wherever we have a single statement in a function we do not need to use curly braces and return, you can just remove curly braces and return. I am not leaving the comment everywhere so please consider the same for other places too.

Reply:
I'll try to find another rule for this. There are too many cases.